source/_integrations/roomba.markdown

```diff
@@ -95,3 +95,13 @@ roomba:
 ### Retrieving your credentials
 
 Please refer to [here](https://github.com/NickWaterton/Roomba980-Python#how-to-get-your-usernameblid-and-password) or [here](https://github.com/koalazak/dorita980#how-to-get-your-usernameblid-and-password) to retrieve both the BLID (username) and the password.
+
+For Home Assistant Container, the following command retrieves the BLID (username) and password:
+
+<div class='note'>
+The command to retrieve the credentials does not need any additional software to be installed because it uses the built-in [roombapy](https://github.com/pschmitt/roombapy) package and [password](https://github.com/pschmitt/roombapy/blob/1.6.1/roomba/entry_points.py#L20) funciton deployed with Home Assistant.
+</div>
```

Comment:
If you had checked the actual output of your change, you would have noticed the Markdown doesn't render.

```suggestion
<div class='note'>

The command to retrieve the credentials does not need any additional software to be installed because it uses the built-in [roombapy](https://github.com/pschmitt/roombapy) package and [password](https://github.com/pschmitt/roombapy/blob/1.6.1/roomba/entry_points.py#L20) function deployed with Home Assistant.

</div>
```

Reply:
@frenck Thanks, for showing me how to fix the rendering problem. I've submitted commit with the fix, and I've verified that the rendering is correct in the netlify preview.